### ISSUE TITLE
Error when running behind a proxy

### ISSUE THREAD

**Issue by advancedeng** [0xd34db33f/maltego-transforms#1]
FYI, 

There are some errors when using these transforms behind a proxy server (that may require authentication as well).

Cheers

**Comment by 0xd34db33f** [comment]
Yeah unfortunately I've never had a situation to test it behind a proxy so this isn't totally unsurprising. Based on the following it should be pretty trivial to add a fix though. I'll see if I can't get a test environment up in the next week or so and fiddle with it a bit.

http://stackoverflow.com/questions/8287628/proxies-with-python-requests-module

**Comment by 0xd34db33f** [comment]
I just uploaded a version that should work with un-authenticated proxies and basic auth proxies. NTLM based proxies are a pain to support in python so I doubt I'll be able to do anything about that for now. I haven't found time to setup a full environment to test with so there might be a bug or two. Give it a whirl if you want and let me know. I'll help fix things if still broken.

**Comment by glerinsonum** [comment]
@0xd34db33f python not better. 

Brother do you have fb ?
